src/components/staffing/ProjectCell.js

```diff
@@ -29,11 +29,22 @@ export default class ProjectCell extends React.Component {
     const upcomingProjects = projects.filter(
       (project) => { return !currentProjects.includes(project) }
     )
-    const currentProjectsText = currentProjects.join(', ')
+    const renamedCurrentProjects = currentProjects.map(
+      (project) => {
+        return MAP_PROJECT_NAMES[project]
+        ? MAP_PROJECT_NAMES[project]
+        : project
+      }
+    )
+    const currentProjectsText = renamedCurrentProjects.join(', ')
     const separator = currentProjects.length > 0 && upcomingProjects.length > 0 ? ', ' : ''
     const upcomingProjectsText = upcomingProjects.join(', ')
     const isIndividualProject = data[rowIndex].projects === undefined
-    const cellText = isIndividualProject ? data[rowIndex][field] : (
+    const individualProjectText = isIndividualProject ? data[rowIndex][field] : ''
+    const individualProjectRenamedText = MAP_PROJECT_NAMES[individualProjectText]
+    ? MAP_PROJECT_NAMES[individualProjectText]
+    : individualProjectText
+    const cellText = isIndividualProject ? individualProjectRenamedText : (
```

Comment:
i know you didnt write this code but would be better if this was a component

Reply:
Fixed!